funtimes.Rproj

```diff
@@ -0,0 +1,13 @@
+Version: 1.0
```

Comment:
Please, exclude this file from the commit, because it is not related to the package.